src/reconcile.rs

```diff
@@ -132,16 +132,20 @@ where
         self.resource.meta().clone()
     }
 
-    /// This lists all Pods that have an OwnerReference that points to us (the object from `self.resource`)
+    /// This lists all Resources that have an OwnerReference that points to us (the object from `self.resource`)
     /// as its Controller.
     ///
     /// Unfortunately the Kubernetes API does _not_ allow filtering by OwnerReference so we have to fetch
-    /// all Pods and filter them on the client.
+    /// all Resources and filter them on the client.
     /// To reduce this overhead a LabelSelector will be included that uses the standard
     /// `app.kubernetes.io/instance` label and will use the name of the resource in this context
     /// as its value.
     /// You need to make sure to always set this label correctly!
```

Comment:
Maybe a hint for labels::get_recommended_labels? And you could use that instead of (further down):
`let mut labels = BTreeMap::new();`
`labels.insert(labels::APP_INSTANCE_LABEL.to_string(), self.resource.name());`

Reply:
Good point, thank you.

Reply:
Well....that fails the build. I'll need to understand what's going on.

Reply:
Take a look please. It seems to build now.